code/modules/hydroponics/seeds.dm

```diff
@@ -79,11 +79,23 @@
 		icon_harvest = "[species]-harvest"
 
 	if(!nogenes)
+		// Bubber edit
```

Comment:
Enclose this in a block because future reviewers will not know where this edit starts and ends